src/core.ts

```diff
@@ -208,6 +227,24 @@ function mapContracts(solcOutput: SolcOutput): ContractMap {
   return res;
 }
 
+function getInternalVariables(contract: ContractDefinition, contractMap: ContractMap): VariableDeclaration[] {
+  const parents = contract.linearizedBaseContracts.map(id => mustGet(contractMap, id));
+
+  const overriden = new Set<number>();
+  const res = [];
+
+  for (const parent of parents) {
+    for (const v of findAll('VariableDeclaration', parent)) {
+      if (v.stateVariable && v.visibility === 'internal' && !overriden.has(v.id)) {
+        res.push(v);
+        overriden.add(v.id);
```

Comment:
Does this make sense?

Reply:
This can be removed. It's used for internal function but it doesn't make sense for variables.

Reply:
This can be removed. It's used for internal function but it doesn't make sense for variables.